src/Components/v2/CountrySelect.tsx

```diff
@@ -13,6 +13,7 @@ export const CountrySelect = (props: CountrySelectProps) => {
 }
 
 const COUNTRY_SELECT_OPTIONS = [
+  { text: "Select a country", value: "" },
```

Comment:
Corresponding PR since this will affect other pages, including those in the OrderApp: https://github.com/artsy/reaction/pull/2708

Reply:
I rather like the approach @ds300 made in [this comment](https://github.com/artsy/reaction/pull/2708#issuecomment-522952081)!